dev/bots/suite_runners/run_framework_tests.dart

```diff
@@ -235,7 +235,11 @@ Future<void> frameworkTestsRunner() async {
       tests: <String>[ 'test' ],
     );
     await runDartTest(path.join(flutterRoot, 'dev', 'bots'));
-    await runDartTest(path.join(flutterRoot, 'dev', 'devicelab'), ensurePrecompiledTool: false); // See https://github.com/flutter/flutter/issues/86209
+    await runDartTest(
+      path.join(flutterRoot, 'dev', 'devicelab'),
+      ensurePrecompiledTool: false,  // See https://github.com/flutter/flutter/issues/86209
+      perTestTimeout: Duration(seconds: 45) // Standard 30 is flaky because of a long running test, https://github.com/flutter/flutter/issues/156456
```

Comment:
```suggestion
      perTestTimeout: const Duration(seconds: 45) // Standard 30 is flaky because of a long running test, https://github.com/flutter/flutter/issues/156456
```